tsconfig.json

```diff
@@ -25,7 +25,8 @@
 
     /* Experimental Options */                
     "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
-    "emitDecoratorMetadata": true          /* Enables experimental support for emitting type metadata for decorators. */
+    "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
+    "strictPropertyInitialization": false,      /* Disables mandatory property initialization */
```

Comment:
Why did you change this config?

Reply:
Not sure it is a good idea to "relax" a TypeScript option just for a test implementation. You cannot initialise them instead?

Reply:
I'll do that instead then